config/namelist.io

```diff
@@ -9,6 +9,7 @@ ldiag_dMOC       =.false.
 ldiag_DVD        =.false.
 ldiag_forc       =.false.
 ldiag_extflds    =.false.
+ldiag_destinE    =.false.
```

Comment:
Would be great to use a name that tells the user what diagnostic will be created.